src/Simulation/Common/QubitManager.cs

```diff
@@ -368,8 +383,8 @@ public void Disable(IQArray<Qubit> qubitsToDisable)
             if (!DisableBorrowing)
             {
                 if (!operationStack.Peek().Locals.TryAdd(ret, 1))
-                {

```

Comment:
Here and in 3 other places. Code seems to be the same, yet it's marked as changed...

Reply:
Yes, line endings in this file weren't consistent. VS Code normalized them to "LF"